js/app/game.js

```diff
@@ -263,6 +263,10 @@ define([
         }
 
         var changeScene = (sceneKey) => {
+            if(!this.scenes[sceneKey]) {
```

Comment:
https://github.com/The-Brains/ClickAndPointLib/blob/0623a4a363da10dac9f88ad41d083857c81aaea3/js/app/game.js#L181-L194

Reply:
That function seems to do too many things. I'd rather do a simple check in scenes.

Reply:
ok